gaia-header.js

```diff
@@ -123,6 +124,7 @@ module.exports = component.register('gaia-header', {
     debug('attached');
     this.runFontFitSoon();
     this.observerStart();
+    window.addEventListener('resize', this);
```

Comment:
That's the pattern I normally use. But I guess it's just personal preference. The `handleEvent` thing is a bit 'global' for my liking :) Although I understand it is somewhat of a convention in Gaia, it tends to lead to a large messy function that does thousands of things.